Makefile

```diff
@@ -153,10 +153,13 @@ endif
 
 TARGET_ARCH_DEBIAN = $(shell ./scripts/build/architecture-convert.sh -r $(TARGET_ARCH) -t debian)
 
+
+TARGET_ARCH_REDHAT = $(shell ./scripts/build/architecture-convert.sh -r $(TARGET_ARCH) -t redhat)
```

Comment:
...and it probably makes sense for these TARGET_ARCH_... to be on consecutive lines, rather than having a blank line between them ;)